### PR TITLE
Reflected real transfer status

### PR DESCRIPTION
**Pull request by DmitriyMolch**
https://cerenetwork.atlassian.net/browse/CBI-2097

### CODE REVIEW

File: src/Contexts/NetworkManagerContext.tsx [hunk 1]
```diff
@@ -56,7 +56,8 @@ export type TransitMessage = {
 
 export type TransactionStatus =
   | "Initializing Transfer"
-  | "In Transit"
+  | "Transfer from Source"
```
Comment (AndreiNavoichyk) [inline src/Contexts/NetworkManagerContext.tsx:59]: Statuses do not look clear. 
Please, draw them on a [diagram](https://miro.com/app/board/uXjVO_9kd3o=/?moveToWidget=3458764529262498598&cot=14) first.

Comment (DmitriyMolch) [reply src/Contexts/NetworkManagerContext.tsx:59]: Don't have permissions to update it on my Cere account.

Comment (AndreiNavoichyk) [reply src/Contexts/NetworkManagerContext.tsx:59]: Granted.

Comment (DmitriyMolch) [reply src/Contexts/NetworkManagerContext.tsx:59]: Updated

Comment (AndreiNavoichyk) [reply src/Contexts/NetworkManagerContext.tsx:59]: Please, add the events when statuses are changing for EVM <-> Cere flows.

Comment (DmitriyMolch) [reply src/Contexts/NetworkManagerContext.tsx:59]: Updated miro diagram with technical details

File: src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx [hunk 2]
```diff
@@ -398,8 +399,8 @@ export const EVMHomeAdaptorProvider = ({
           (destChainId, resourceId, depositNonce, tx) => {
             setHomeTransferTxHash(tx.transactionHash);
             setDepositNonce(`${depositNonce.toString()}`);
-            setTransactionStatus("In Transit");
-            analytics.trackTransferInTransitEvent({
+            setTransactionStatus("Transfer to Destination ");
```
Comment (AndreiNavoichyk) [inline src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx]: Remove extra space

File: src/Contexts/Adaptors/SubstrateHomeAdaptor.tsx [hunk 3]
```diff
@@ -268,8 +279,8 @@ export const SubstrateHomeAdaptorProvider = ({
                       const depositNonce = `${response.toJSON()}`;
                       setDepositNonce(depositNonce);
                       setHomeTransferTxHash(status.asFinalized.toHex());
-                      setTransactionStatus("In Transit");
-                      analytics.trackTransferInTransitEvent({
+                      setTransactionStatus("Transfer to Destination ");
```
Comment (AndreiNavoichyk) [inline src/Contexts/Adaptors/SubstrateHomeAdaptor.tsx]: Remove extra space

File: src/Services/Analytics.ts [hunk 4]
```diff
@@ -26,13 +26,25 @@ class AnalyticsService {
     this.ga = new GA(options.ga);
```
Comment (AndreiNavoichyk) [inline src/Services/Analytics.ts:26]: Add `transfer_initialized` event and use it.

File: src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx (hunk 2 above)
Comment (DmitriyMolch) [reply src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx]: fixed

File: src/Contexts/Adaptors/SubstrateHomeAdaptor.tsx (hunk 3 above)
Comment (DmitriyMolch) [reply src/Contexts/Adaptors/SubstrateHomeAdaptor.tsx]: fixed

File: src/Services/Analytics.ts (hunk 4 above)
Comment (DmitriyMolch) [reply src/Services/Analytics.ts:26]: fixed to `transfer_initializing` to reflect current application  logic

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**